Topic0Ejercicio3/src/building.java

```diff
@@ -0,0 +1,9 @@
+
+public abstract class building {
```

Comment:
Class names should be nouns, in mixed case with the first letter of each internal word capitalized